_sass/typography.scss

```diff
@@ -0,0 +1,60 @@
+/** Styles and fonts copied from https://www.ginger.io/brand */
+
+@font-face {
+  font-family: "Heldane Display";
+  src: url(/fonts/Heldane_Display_Regular.otf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Heldane Display";
+  src: url(/fonts/Heldane_Display_Bold.otf);
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Untitled Sans";
+  src: url(/fonts/Untitled_Sans_Regular.ttf);
+  font-weight: bold;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: "Heldane display", sans-serif;
+}
+
+h1 {
+  margin: 0.67em 0;
+  margin-top: 0px;
+  margin-bottom: 25px;
+  font-size: 96px;
```

Comment:
We'll likely want to make the font size smaller...headers are HUUUUGE on the branding website :).